2018-01-03 10:18:50 ERROR - no sufficient fund to transfer 
com.sapient.exceptions.BalException: no sufficient fund to transfer 
	at com.sapient.service.SerImpl.transferFund(SerImpl.java:41)
	at com.sapient.client.TransferClient.main(TransferClient.java:31)
2018-01-03 10:18:50 INFO - transfer the amount rs 5000 from account ID 2003 to 1002
2018-01-03 10:18:50 TRACE - transferfund() invoked in service
2018-01-03 10:18:50 TRACE - gwt account method invoked
2018-01-03 10:18:50 DEBUG - checking the aid against the map key
2018-01-03 10:18:50 ERROR - no id found
com.sapient.exceptions.IdException: no id found
	at com.sapient.dao.DaoImpl.getAccount(DaoImpl.java:31)
	at com.sapient.service.SerImpl.transferFund(SerImpl.java:35)
	at com.sapient.client.TransferClient.main(TransferClient.java:39)
2018-01-03 10:19:00 TRACE - main thred starts
2018-01-03 10:19:00 TRACE - service constructor invoked
2018-01-03 10:19:00 TRACE - static block called in dao
2018-01-03 10:19:00 DEBUG - hashmap initialised using three account instances
2018-01-03 10:19:00 TRACE - setter setDao() called
2018-01-03 10:19:00 DEBUG - injecting dao
2018-01-03 10:19:00 INFO - dao initialized
2018-01-03 10:19:00 INFO - service instance is ready with dao dependency
2018-01-03 10:19:00 DEBUG - service instance created
2018-01-03 10:19:00 INFO - transfer the amount rs 5000 from account ID 1001 to 1002
2018-01-03 10:19:00 TRACE - transferfund() invoked in service
2018-01-03 10:19:00 TRACE - gwt account method invoked
2018-01-03 10:19:00 DEBUG - checking the aid against the map key
2018-01-03 10:19:00 INFO - account ID 1001 found
2018-01-03 10:19:00 DEBUG - from account retrieved
2018-01-03 10:19:00 TRACE - gwt account method invoked
2018-01-03 10:19:00 DEBUG - checking the aid against the map key
2018-01-03 10:19:00 INFO - account ID 1002 found
2018-01-03 10:19:00 DEBUG - to account retrieved
2018-01-03 10:19:00 INFO - 1001 and 1002 accounts are updates successfully
2018-01-03 10:19:00 INFO - successfully transfered
2018-01-03 10:19:00 INFO - transfer the amount rs 5000 from account ID 1003 to 1002
2018-01-03 10:19:00 TRACE - transferfund() invoked in service
2018-01-03 10:19:00 TRACE - gwt account method invoked
2018-01-03 10:19:00 DEBUG - checking the aid against the map key
2018-01-03 10:19:00 INFO - account ID 1003 found
2018-01-03 10:19:00 DEBUG - from account retrieved
2018-01-03 10:19:00 TRACE - gwt account method invoked
2018-01-03 10:19:00 DEBUG - checking the aid against the map key
2018-01-03 10:19:00 INFO - account ID 1002 found
2018-01-03 10:19:00 DEBUG - to account retrieved
2018-01-03 10:19:00 ERROR - no sufficient fund to transfer 
com.sapient.exceptions.BalException: no sufficient fund to transfer 
	at com.sapient.service.SerImpl.transferFund(SerImpl.java:41)
	at com.sapient.client.TransferClient.main(TransferClient.java:31)
2018-01-03 10:19:00 INFO - transfer the amount rs 5000 from account ID 2003 to 1002
2018-01-03 10:19:00 TRACE - transferfund() invoked in service
2018-01-03 10:19:00 TRACE - gwt account method invoked
2018-01-03 10:19:00 DEBUG - checking the aid against the map key
2018-01-03 10:19:00 ERROR - no id found
com.sapient.exceptions.IdException: no id found
	at com.sapient.dao.DaoImpl.getAccount(DaoImpl.java:31)
	at com.sapient.service.SerImpl.transferFund(SerImpl.java:35)
	at com.sapient.client.TransferClient.main(TransferClient.java:39)
2018-01-03 10:19:03 TRACE - main thred starts
2018-01-03 10:19:03 TRACE - service constructor invoked
2018-01-03 10:19:03 TRACE - static block called in dao
2018-01-03 10:19:03 DEBUG - hashmap initialised using three account instances
2018-01-03 10:19:03 TRACE - setter setDao() called
2018-01-03 10:19:03 DEBUG - injecting dao
2018-01-03 10:19:03 INFO - dao initialized
2018-01-03 10:19:03 INFO - service instance is ready with dao dependency
2018-01-03 10:19:03 DEBUG - service instance created
2018-01-03 10:19:03 INFO - transfer the amount rs 5000 from account ID 1001 to 1002
2018-01-03 10:19:03 TRACE - transferfund() invoked in service
2018-01-03 10:19:03 TRACE - gwt account method invoked
2018-01-03 10:19:03 DEBUG - checking the aid against the map key
2018-01-03 10:19:03 INFO - account ID 1001 found
2018-01-03 10:19:03 DEBUG - from account retrieved
2018-01-03 10:19:03 TRACE - gwt account method invoked
2018-01-03 10:19:03 DEBUG - checking the aid against the map key
2018-01-03 10:19:03 INFO - account ID 1002 found
2018-01-03 10:19:03 DEBUG - to account retrieved
2018-01-03 10:19:03 INFO - 1001 and 1002 accounts are updates successfully
2018-01-03 10:19:03 INFO - successfully transfered
2018-01-03 10:19:03 INFO - transfer the amount rs 5000 from account ID 1003 to 1002
2018-01-03 10:19:03 TRACE - transferfund() invoked in service
2018-01-03 10:19:03 TRACE - gwt account method invoked
2018-01-03 10:19:03 DEBUG - checking the aid against the map key
2018-01-03 10:19:03 INFO - account ID 1003 found
2018-01-03 10:19:03 DEBUG - from account retrieved
2018-01-03 10:19:03 TRACE - gwt account method invoked
2018-01-03 10:19:03 DEBUG - checking the aid against the map key
2018-01-03 10:19:03 INFO - account ID 1002 found
2018-01-03 10:19:03 DEBUG - to account retrieved
2018-01-03 10:19:03 ERROR - no sufficient fund to transfer 
com.sapient.exceptions.BalException: no sufficient fund to transfer 
	at com.sapient.service.SerImpl.transferFund(SerImpl.java:41)
	at com.sapient.client.TransferClient.main(TransferClient.java:31)
2018-01-03 10:19:03 INFO - transfer the amount rs 5000 from account ID 2003 to 1002
2018-01-03 10:19:03 TRACE - transferfund() invoked in service
2018-01-03 10:19:03 TRACE - gwt account method invoked
2018-01-03 10:19:03 DEBUG - checking the aid against the map key
2018-01-03 10:19:03 ERROR - no id found
com.sapient.exceptions.IdException: no id found
	at com.sapient.dao.DaoImpl.getAccount(DaoImpl.java:31)
	at com.sapient.service.SerImpl.transferFund(SerImpl.java:35)
	at com.sapient.client.TransferClient.main(TransferClient.java:39)
2018-01-03 10:19:06 TRACE - main thred starts
2018-01-03 10:19:06 TRACE - service constructor invoked
2018-01-03 10:19:06 TRACE - static block called in dao
2018-01-03 10:19:06 DEBUG - hashmap initialised using three account instances
2018-01-03 10:19:06 TRACE - setter setDao() called
2018-01-03 10:19:06 DEBUG - injecting dao
2018-01-03 10:19:06 INFO - dao initialized
2018-01-03 10:19:06 INFO - service instance is ready with dao dependency
2018-01-03 10:19:06 DEBUG - service instance created
2018-01-03 10:19:06 INFO - transfer the amount rs 5000 from account ID 1001 to 1002
2018-01-03 10:19:06 TRACE - transferfund() invoked in service
2018-01-03 10:19:06 TRACE - gwt account method invoked
2018-01-03 10:19:06 DEBUG - checking the aid against the map key
2018-01-03 10:19:06 INFO - account ID 1001 found
2018-01-03 10:19:06 DEBUG - from account retrieved
2018-01-03 10:19:06 TRACE - gwt account method invoked
2018-01-03 10:19:06 DEBUG - checking the aid against the map key
2018-01-03 10:19:06 INFO - account ID 1002 found
2018-01-03 10:19:06 DEBUG - to account retrieved
2018-01-03 10:19:06 INFO - 1001 and 1002 accounts are updates successfully
2018-01-03 10:19:06 INFO - successfully transfered
2018-01-03 10:19:06 INFO - transfer the amount rs 5000 from account ID 1003 to 1002
2018-01-03 10:19:06 TRACE - transferfund() invoked in service
2018-01-03 10:19:06 TRACE - gwt account method invoked
2018-01-03 10:19:06 DEBUG - checking the aid against the map key
2018-01-03 10:19:06 INFO - account ID 1003 found
2018-01-03 10:19:06 DEBUG - from account retrieved
2018-01-03 10:19:06 TRACE - gwt account method invoked
2018-01-03 10:19:06 DEBUG - checking the aid against the map key
2018-01-03 10:19:06 INFO - account ID 1002 found
2018-01-03 10:19:06 DEBUG - to account retrieved
2018-01-03 10:19:06 ERROR - no sufficient fund to transfer 
com.sapient.exceptions.BalException: no sufficient fund to transfer 
	at com.sapient.service.SerImpl.transferFund(SerImpl.java:41)
	at com.sapient.client.TransferClient.main(TransferClient.java:31)
2018-01-03 10:19:06 INFO - transfer the amount rs 5000 from account ID 2003 to 1002
2018-01-03 10:19:06 TRACE - transferfund() invoked in service
2018-01-03 10:19:06 TRACE - gwt account method invoked
2018-01-03 10:19:06 DEBUG - checking the aid against the map key
2018-01-03 10:19:06 ERROR - no id found
com.sapient.exceptions.IdException: no id found
	at com.sapient.dao.DaoImpl.getAccount(DaoImpl.java:31)
	at com.sapient.service.SerImpl.transferFund(SerImpl.java:35)
	at com.sapient.client.TransferClient.main(TransferClient.java:39)
2018-01-03 10:19:09 TRACE - main thred starts
2018-01-03 10:19:09 TRACE - service constructor invoked
2018-01-03 10:19:09 TRACE - static block called in dao
2018-01-03 10:19:09 DEBUG - hashmap initialised using three account instances
2018-01-03 10:19:09 TRACE - setter setDao() called
2018-01-03 10:19:09 DEBUG - injecting dao
2018-01-03 10:19:09 INFO - dao initialized
2018-01-03 10:19:09 INFO - service instance is ready with dao dependency
2018-01-03 10:19:09 DEBUG - service instance created
2018-01-03 10:19:09 INFO - transfer the amount rs 5000 from account ID 1001 to 1002
2018-01-03 10:19:09 TRACE - transferfund() invoked in service
2018-01-03 10:19:09 TRACE - gwt account method invoked
2018-01-03 10:19:09 DEBUG - checking the aid against the map key
2018-01-03 10:19:09 INFO - account ID 1001 found
2018-01-03 10:19:09 DEBUG - from account retrieved
